ranger-plugin/conf/ranger-servicedef-hawq.json

```diff
@@ -244,7 +244,7 @@
       "name": "authentication",
       "type": "enum",
       "subType": "authType",
-      "mandatory": false,
+      "mandatory": true,
```

Comment:
Yes, we aim to update the connection to HAWQ using Kerberos when Kerberos authentication is set in pg_hba.conf. 
Why don't Ambari change pg_hba.conf in HAWQ once Kerberos is setup? I think we need protect connection to HAWQ.